ntfy/cli.py

```diff
@@ -49,9 +49,10 @@ def send_notification(message, args, config):
         module = import_module('ntfy.backends.{}'.format(backend))
 
         try:
+            backend_config = config.get(backend, {})
+            backend_config.update(args.option)
```

Comment:
Regarding config, I still vote to allow multiple configs per backend, (not tie the config key to the module itself but have the module a required key).  So that the thing I have to type on the command like can be WTF I want, rather than having to know "xmpp".   Perhaps my method of contacting tara's phone has changed - I don't want to have to edit all my ntfy-using-scripts to reflect that - just update the config item for 'tara-phone'.  And of course that I can configure multiple things per backend via config and not have to override things like passwords in places like command lines :-P